### PR TITLE
passwordless ssh for vios

### PR DESCRIPTION
**Pull request by smruti77**
passwordless ssh vios for network_virtualization, veth_dlpar and
dlpar_vscsi test.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
[dlpar_vscsi.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4688352/dlpar_vscsi.log)
[network_virtualization.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4688355/network_virtualization.log)
[veth_dlpar.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4688356/veth_dlpar.log)

**Comment by smruti77** [comment]
@abdhaleegit review please.

File: io/disk/dlpar_vscsi.py [hunk 1]
```diff
@@ -57,15 +38,16 @@ def setUp(self):
         self.vios_ip = self.params.get('vios_ip', '*', default=None)
         self.vios_user = self.params.get('vios_username', '*', default=None)
         self.vios_pwd = self.params.get('vios_pwd', '*', default=None)
-        self.login(self.vios_ip, self.vios_user, self.vios_pwd)
+        self.session = Session(self.vios_ip, user=self.vios_user,
+                               password=self.vios_pwd)
```
Comment (abdhaleegit) [inline io/disk/dlpar_vscsi.py:42]: So in all the test cases, does it require to close this session ?  should this be taken care in teardown ?

Comment (smruti77) [reply io/disk/dlpar_vscsi.py:42]: not needed

Comment (smruti77) [reply io/disk/dlpar_vscsi.py:42]: not needed this.

**Comment by abdhaleegit** [comment]
@narasimhan-v I ACK, you can merge this

**Pull request merged**